doc/twitter-bootstrap.txt

```diff
@@ -0,0 +1,10 @@
+Bootstrap
+=========
+
+We use Twitter's Open Source Bootstrap CSS framework to ensure our content
+displays correctly on mobiles.
+
+Generation
+----------
+
+See `contrib/bootstrap` for more information on this.
```

Comment:
Are you planning on adding more things here later? In its current status i don't see why we need this.